db/migrate/20180726142030_create_physical_disk.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePhysicalDisk < ActiveRecord::Migration[5.0]
+  def change
+    create_table :physical_disks do |t|
+      t.string :model
+      t.string :vendor
+      t.string :status
+      t.string :location
+      t.index  :location
+      t.string :serial_number
+      t.string :health_state
+      t.string :type
```

Comment:
@EsdrasVP what is this `type` column? It seems it's not related to definition of STI subclass, but some custom value...
When I ran specs, there is for example "SAS" value

Reply:
Ah you're right I assumed it was going to be used for STI.  We'll need to rename this something like disk_type or actually do STI

Reply:
> Ah you're right I assumed it was going to be used for STI. We'll need to rename this something like disk_type or actually do STI

I think we can go for `disk_type` then, since this has nothing to do with existing STI classes.